src/PinAssignment/PortsModel.cpp

```diff
@@ -26,7 +26,7 @@ PortsModel::PortsModel(QObject *parent)
     : QObject(parent), m_model(new QStringListModel(this)) {}
 
 QStringList PortsModel::headerList() const {
-  return {"Name", "Dir", "Package Pin", "Type", "Range"};
+  return {"Name", "Dir", "Package Pin", "Type"};
```

Comment:
Don't we need range anymore ?

Reply:
I don't think so. This is an information for build buses. And it is reflected in the table.

Reply:
Seems like we need "Range" after speaking to Tao. Let's connect tomorrow to discuss offline.